File: Service/src/main/java/com/github/spb/tget/demo/repository/dbRepository/ContactInformationDbRepository.java
package com.github.spb.tget.demo.repository.dbRepository;

public class ContactInformationDbRepository extends DbRepository {

    private ContactInformationDbRepository() {
        this.entityName = "ContactInformation";
    }

    public static ContactInformationDbRepository create() {
        return new ContactInformationDbRepository();
    }
}

File: Service/src/main/java/com/github/spb/tget/demo/repository/dbRepository/ContactInformationDbRepository.kt
package com.github.spb.tget.demo.repository.dbRepository

import com.github.spb.tget.demo.data.ContactEntity

class ContactInformationDbRepository private constructor() : DbRepository<ContactEntity>() {

    init {
        this.entityName = "ContactEntity"
    }

    companion object {

        fun create(): ContactInformationDbRepository {
            return ContactInformationDbRepository()
        }
    }
}
